apps/backend/src/api/devices.py

```
@@ -153,3 +168,79 @@ async def get_device_summary_by_hostname(
+@router.get("/{hostname}/metrics")
+async def get_device_metrics_by_hostname(
+    hostname: str = Path(..., description="Device hostname"),
+    include_processes: bool = Query(False, description="Include process information"),
+    timeout: int = Query(60, description="SSH timeout in seconds"),
+    current_user=Depends(get_current_user)
+):
+    """Get comprehensive system performance metrics from a device"""
+    # TODO: This endpoint needs to be updated to use the new granular monitoring functions
+    # or removed if deprecated. The get_device_info function is no longer available.
+    raise HTTPException(
+        status_code=501, 
```

Ruff (0.12.2)

Trailing whitespace

Remove trailing whitespace

(W291)